### PR TITLE
usb: mass_storage: rework and cleanup logging

### PR DESCRIPTION
**Pull request by jfischer-no**
This patch reworked logging so that the logging levels are more
finely graded and the locations with non-critical error handling
(below usb_ep_set_stall's) are logged with SYS_LOG_WRN.
Also cleanup, add log domain and fix indents.

### CODE REVIEW

File: subsys/usb/class/mass_storage.c [hunk 1]
```diff
@@ -481,7 +482,7 @@ static void CBWDecode(u8_t *buf, u16_t size)
 				} else {
 					usb_ep_set_stall(
 					    CONFIG_MASS_STORAGE_IN_EP_ADDR);
-					SYS_LOG_DBG("BI-STALL");
+					SYS_LOG_WRN("BI-STALL");
```
Comment (nashif) [inline subsys/usb/class/mass_storage.c]: if this is going to be a warning, we need a proper warning message, what is the warning here?

File: subsys/usb/class/mass_storage.c [hunk 2]
```diff
@@ -466,7 +467,7 @@ static void CBWDecode(u8_t *buf, u16_t size)
 				} else {
 					usb_ep_set_stall(
 					    CONFIG_MASS_STORAGE_OUT_EP_ADDR);
-					SYS_LOG_DBG("BO-STALL");
+					SYS_LOG_WRN("BO-STALL");
```
Comment (nashif) [inline subsys/usb/class/mass_storage.c]: if this is going to be a warning, we need a proper warning message, what is the warning here?

**Review by nashif — CHANGES_REQUESTED** [review]

File: subsys/usb/class/mass_storage.c (hunk 2 above)
Comment (jfischer-phytec-iot) [reply subsys/usb/class/mass_storage.c]: It says that the bulk out endpoint is stalled. Longer logs would affect the behavior even more as it is called here in the ISR context (major issue from the usb stack which needs to be fixed).

**Review by finikorg — APPROVED** [review]
Although I think warnings might be improved as Anas mentioned.

**Review by nashif — APPROVED** [review]

**Pull request merged**